### PR TITLE
Port to create fabric -f release

### LINKED ISSUE #33: [1.20.1] Update to Create 0.5.1-f
Fabric version with Create 0.5.1-f doesn't load with
```
---- Minecraft Crash Report ----
// I bet Cylons wouldn't have this problem.

Time: 2024-03-09 16:47:58
Description: Initializing game

java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'creategoggles'!
	at net.fabricmc.loader.impl.FabricLoaderImpl.lambda$invokeEntrypoints$2(FabricLoaderImpl.java:388)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:386)
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:52)
	at net.minecraft.class_310.<init>(class_310.java:458)
	at net.minecraft.client.main.Main.main(Main.java:211)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:243)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:278)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)
Caused by: java.lang.NoSuchMethodError: 'com.simibubi.create.foundation.data.CreateRegistrate com.simibubi.create.foundation.data.CreateRegistrate.useCreativeTab(net.minecraft.class_5321)'
	at com.robocraft999.creategoggles.fabric.CreateGogglesFabric.onInitialize(CreateGogglesFabric.java:13)
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:384)
	... 14 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.fabricmc.loader.impl.FabricLoaderImpl.lambda$invokeEntrypoints$2(FabricLoaderImpl.java:388)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:386)
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:52)
	at net.minecraft.class_310.<init>(class_310.java:458)

-- Initialization --
Details:
	Modules: 
Stacktrace:
	at net.minecraft.client.main.Main.main(Main.java:211)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:243)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:278)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)

```
Create version: `create-fabric-0.5.1-f-build.1335+mc1.20.1`
CreateGoggles version: `creategoggles-1.20.1-0.5.5.f-[FABRIC]`

With Create version  `create-fabric-0.5.1-d-build.1161+mc1.20.1` minecraft runs without problems

### PR DESCRIPTION
**Pull request by solonovamax**
Fixes #33

### CODE REVIEW

**Comment by raftario** [comment]
@Robocraft999 Any chance to get a new release soon to fix this ?

File: fabric/src/main/java/com/robocraft999/creategoggles/fabric/CreateGogglesFabric.java [hunk 1]
```diff
@@ -10,7 +10,7 @@
 public class CreateGogglesFabric implements ModInitializer {
     @Override
     public void onInitialize() {
-        CreateGoggles.REGISTRATE.useCreativeTab(CreativeModeTabs.COMBAT);
+        CreateGoggles.REGISTRATE.setCreativeTab(CreativeModeTabs.COMBAT);
```
Comment (Robocraft999) [inline fabric/src/main/java/com/robocraft999/creategoggles/fabric/CreateGogglesFabric.java:13]: actually should use defaultCreativeTab

**Review by Robocraft999 — COMMENTED** [review]
I fixed this first in 1.19.2 and then merge it into 1.20.1 so don't be surprised that I'm closing this now

**Comment by solonovamax** [comment]
> I fixed this first in 1.19.2 and then merge it into 1.20.1 so don't be surprised that I'm closing this now

ya no worries

tbh I did it mainly for myself so I can use it, and just made a pr with it bc why not

**Pull request closed without merging**